bin/p2-preparer/main.go

```diff
@@ -1,21 +1,27 @@
 package main
 
 import (
+	"io/ioutil"
 	"os"
 	"os/signal"
+	"strings"
 	"sync"
 	"syscall"
+	"time"
 
 	"github.com/Sirupsen/logrus"
 	"gopkg.in/alecthomas/kingpin.v2"
 
 	"github.com/square/p2/pkg/logging"
 	"github.com/square/p2/pkg/preparer"
+	"github.com/square/p2/pkg/types"
 	"github.com/square/p2/pkg/util/param"
 	"github.com/square/p2/pkg/version"
 	"github.com/square/p2/pkg/watch"
 )
 
+const P2WHITELIST_CONFIGCHECK_INTERVAL = 10 * time.Second
```

Comment:
Can you move this to `pkg/constants/constants.go`

Reply:
Also, what led you to settle on 10 seconds for the interval out of curiosity?

Reply:
And, see https://github.com/golang/go/wiki/CodeReviewComments#mixed-caps for const naming convention.

Reply:
I think it does not need to check every second or two, and 1 min may be too long if user already sent signal to let p2-prepare proceed. Let me know if you have a better one.